recipes/conversation_view/feed_pill/feed_item_pill.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div :class="[wrapperClass, expanded ? 'dt-feed-item--expanded-pill' : 'd-bar-pill']">
+    <div :class="['d-bgc-secondary', expanded ? 'dt-feed-item--expanded-pill' : 'd-bar-pill']">
+      <div class="d-p8">
+        <button
```

Comment:
Is there really supposed to be no styling on hover? I notice there also isn't in the prototype you posted but it kinda feels wrong to me... verify with design?

Reply:
confirmed lol

Reply:
okay haha